kaitaistruct.py

```diff
@@ -1,18 +1,19 @@
 import itertools
 import sys
 import struct
+import zlib
 from io import open, BytesIO, SEEK_CUR, SEEK_END  # noqa
 
-PY2 = sys.version_info[0] == 2
+PY_OLD = sys.version_info[0] == 2 or (sys.version_info[0] == 3 and sys.version_info[1] < 2)
 
-# Kaitai Struct runtime streaming API version, defined as per PEP-0396
-# standard. Used for two purposes:
+# Kaitai Struct runtime streaming API version, defined as per PEP-0396 standard.
```

Comment:
Please no formatting-only changes. If you want to bring them, they must be in a separate PR with bulletproof reasoning why they are needed.

Reply:
Done. I've reverted them.